modules/conversion-center/module.php

```diff
@@ -45,6 +47,13 @@ public static function is_active(): bool {
 		return Plugin::$instance->experiments->is_feature_active( static::EXPERIMENT_NAME );
 	}
 
+	public static function is_pro_active(): bool {
```

Comment:
@louiswol94 I am sure there is a bebtter way but couldn't find it.

Reply:
You can use `Utils::has_pro()`